awx/main/tests/functional/api/test_inventory.py

```diff
@@ -499,7 +499,7 @@ def test_vault_credential_not_allowed_via_related(self, project, inventory, vaul
         assert 'Credentials of type insights and vault' in r.data['msg']
         assert 'disallowed for scm inventory sources' in r.data['msg']
 
-    def test_credentials_relationship_mapping(self, project, inventory, organization, admin_user, post, patch):
+    def test_credentials_relationship_mapping(self, project, inventory, organization, admin_user, post, patch, test_credentials_relationship_mapping):
```

Comment:
Looks like a mistake?

Reply:
Yes, thank you so much!  This has been reverted here:[ f9a2f43e4ad3f706a942e326119258373c247390](https://github.com/ansible/awx/pull/15537/commits/f9a2f43e4ad3f706a942e326119258373c247390)

Reply:
ack